docs/usage.md

```diff
@@ -50,8 +52,208 @@ The following matrices are displayed by the SOC Management Widget:
   **Percentage Value**
   The Percentage value is the difference between the time period given while configuration and the the data prior to the number of days given while configuration.  
   eg. For ROI if the date is 20/12/2022 and the widget is configured for 10 days then the percentage is for the (ROI obtained from 10/12-20/12) - (ROI obtained from 1/12-9/12) .
-  
-  Known behaviour:
+
+    Known behaviour:
     1. If no data is present for the time period before for which the widget has been configured, then the percentage increase will be 100%.
     2. If a playbook has execution history but has been deleted and comes into the top3playbooks box then the hyperlink will redirect to "Page not found".
+<hr>
+
+  # Data Source :  'Record Containing JSON Data' Option
+
+  Use this option to populate the widget using a json.
+  The Json object must be present as a record in any of the modules.
+  
+  JSON data:
+
+  Format to follow in the json to populate the data seamlessly

```

Comment:
Please see below for expected JSON format to populate the widget data seamlessly